src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -162,14 +162,6 @@ internal override ImmutableArray<NamedTypeSymbol> GetInterfacesToEmit()
             throw ExceptionUtilities.Unreachable;
         }
 
-        protected sealed override ImmutableArray<NamedTypeSymbol> MakeAllInterfaces()
-        {
-            // Because declared types will have been checked for "uniqueness of implemented interfaces" (C# 4 spec, 13.4.2),
-            // we are guaranteed that none of these substitutions collide in a correct program.  Consequently, we can simply
-            // substitute the original interfaces.
-            return _unbound ? ImmutableArray<NamedTypeSymbol>.Empty : Map.SubstituteNamedTypes(OriginalDefinition.AllInterfacesNoUseSiteDiagnostics);
```

Comment:
We should make sure VB and C# behavior is consistent here. It doesn't look like it was prior to the change.

---
In reply to: [91140177](https://github.com/dotnet/roslyn/pull/15594#discussion_r91140177) [](ancestors = 91140177)